src/shared/components/LomFieldsInput/LomFieldsInput.tsx

```diff
@@ -20,9 +20,10 @@ import {
 type LomFieldsInputProps = {
 	loms: Avo.Lom.LomField[];
 	onChange: (newLoms: Avo.Lom.LomField[]) => void;
+	isAssignment?: boolean;
```

Comment:
Let's call this showThemas since this component doesn't need to know about assignments
```suggestion
	showThemas?: boolean;
```